api/src/main/java/org/hawkular/apm/api/model/trace/Node.java

```diff
@@ -213,6 +218,20 @@ public Node setFaultDescription(String faultDescription) {
     }
 
     /**
+     * @return the properties
+     */
+    public Set<Property> getProperties() {
+        return properties;
+    }
+
+    /**
+     * @param properties the properties to set
+     */
+    public void setProperties(Set<Property> properties) {
```

Comment:
Also, the caller is able to set the properties to `null`. If that's acceptable, can you add a simple Javadoc stating this fact?